### ISSUE TITLE
Tab attribute "ParentClassName" is deprecated. You must use "parent_class_name" instead.

### ISSUE THREAD

**Issue by dsimunic** [Invertus/saferpayofficial#115]
This warning appears in prestashop log after installing the latest release, or any of the previous releases.

**Comment by justelis22** [comment]
Hello there @dsimunic,
Thank you for reporting the issue!

Could you please send me your environment (PS and PHP versions)?
Also, steps to recreate the issue would be needed :)

Waiting for your reply!

Best Regards,
Invertus Support team.

**Comment by dsimunic** [comment]
PHP 7.2, PrestaShop 1.7.8.8

Repro:

* download the zip file from the releases page;
* open PrestaShop modules page;
* upload module zip file;
* observe PrestaShop log.

Apologies for not providing that info earlier.

**Comment by justelis22** [comment]
Thanks @dsimunic!

We will check this issue and get back to you as soon as we have some updates!

Best Regards,
Invertus Support team.

**Comment by justelis22** [comment]
@dsimunic,

I have created the exact same environment as you mentioned, but no errors or warnings can be seen from my side.
Could you please contact us via support@invertus.eu email providing access to your shop so that we could debug the issue?
It would be helpful if you could send us BO + FTP credentials.

Waiting for your reply!

Best Regards,
Invertus Support team.

**Comment by dsimunic** [comment]
Thanks, that won't be necessary. I reported this for the benefit of future users of the module.

Release 1.1.2 still tests for `ParentClassName`, though I don't think that is what caused the warning as I see in the history that you fixed the installer a few months ago.

https://github.com/Invertus/saferpayofficial/blob/master/src/Install/Installer.php#L121

            if (!$this->installTab($tab['class_name'], $tab['ParentClassName'], $tab['name'])) {
                return false;
            }

Can't re-test right now as something got messed up (doesn't let me uninstall the module), so not sure what exactly caused the warnings. Maybe I mistakenly used an earlier release without noticing?

I'll re-check when I set up a testing copy. Thanks.
